style.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  margin: 5px;
+  padding: 5px;
+}
+
+.Logo h4 {
+  font-size: 20px;
+  color: #6070ff;
+  margin: 5px;
+  padding: 10px;
+}
+
+.Menu-bar {
+  float: right;
+  position: absolute;
+  top: 10px;
+  right: 20px;
+}
+
+.Menu-bar li {
+  list-style: none;
+  width: 20px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 3px;
+}
+
```

Comment:
- [ ] [optional] Use flex-box alignment for menu-bar and use align-content; centre; for header to align them properly
<img width="551" alt="Screenshot 2023-02-15 at 4 43 49 AM" src="https://user-images.githubusercontent.com/44798044/218888199-97c7557f-df0d-44e5-9452-cb6dad5ab37b.png">